lib/options.js

```diff
@@ -173,19 +173,26 @@ exports.parse = function parse(argv) {
   } else {
     throw new ExitError(fmt('Invalid cluster option: %s', cluster));
   }
-
-  if (options.channel && url.parse(options.channel).protocol) {
+  var protocol;
+  // The protocol includes the colon on parsing!
+  var protocolMap = {
+    'http:': 'http',
+    'https:': 'http',
+    'ws:': 'ws',
+    'wss:': 'wss',
```

Comment:
this works? mapping the `:` variant to the non-`:` variant? if it does, fine by me